pkg/operator/ceph/object/spec.go

```diff
@@ -401,8 +418,14 @@ func (c *clusterConfig) makeDaemonContainer(rgwConfig *rgwConfig) (v1.Container,
 		container.VolumeMounts = append(container.VolumeMounts, mount)
 	}
 	if c.store.Spec.Gateway.CaBundleRef != "" {
-		updatedBundleMount := v1.VolumeMount{Name: caBundleUpdatedVolumeName, MountPath: caBundleExtractedDir, ReadOnly: true}
+		updatedBundleMount := v1.VolumeMount{Name: caBundleUpdatedVolumeName, MountPath: "/etc/ssl/combined-ca", ReadOnly: true}
```

Comment:
Is it possible to use the same bundle mount path in both the init container and RGW container? That seems like a good way to make debugging slightly easier.

Reply:
I will do it. Thank you for your suggestion

Reply:
done